### PR TITLE
documented requirements on the contents of the UserAction, UserAction event types and UserLogin audit event

### PR DESCRIPTION
**Pull request by hnadiminti-equinix**
Desc:
audito-maldito: Formalize requirements for audit event types

[SCT-326](https://packet.atlassian.net/browse/SCT-326)

### CODE REVIEW

**Comment by JAORMX** [comment]
This documentation is useful, but doesn't really do what the PR description describes. Can you rephrase the PR description to match what was done in the PR? I think this can be merged.

**Comment by hnadiminti-equinix** [comment]
Update description as requested ...Thanks @JAORMX for review!

File: processors/auditd/sessiontracker/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Session Tracking Using Audito Maldito.
```
Comment (stephen-fox) [inline processors/auditd/sessiontracker/README.md:1]: Normally I find these Codacy warnings to be useless - but in this case they are generally correct.

- Section headings do not require punctuation (in other words, you do not need to end them with a period)
- Section headings and code fences (the three backtick characters) should have an empty newline between them. For example:

```
## Some stuff

Hello world, this is the some stuff section.

'''
This is a code fence (I used single quotes here
so I do not need to escape them)
'''

## Another section

Here is another section.
```

Note how the section headings and the code fences are separated by empty newlines.

**Review by stephen-fox — CHANGES_REQUESTED** [review]

**Review by stephen-fox — APPROVED** [review]
